### PR TITLE
added pip install dsdk

### PR DESCRIPTION
**Pull request by GrayEye**

### CODE REVIEW

File: {{cookiecutter.name}}/requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-dsdk[psycopg2,pymssql]@git+https://github.com/pennsignals/dsdk.git@1.5.6#egg=dsdk
```
Comment (jlubken) [inline {{cookiecutter.name}}/requirements.txt:1]: This still needs dsdk in jupyterlab.

File: {{cookiecutter.name}}/setup.py [hunk 2]
```diff
@@ -5,11 +5,7 @@
 from setuptools import find_packages, setup
 
 INSTALL_REQUIRES = (
-    (
-        "dsdk[psycopg2,pymssql]@"
-        "git+https://github.com/pennsignals/dsdk.git"
-        "@1.5.6#egg=dsdk"
-    ),
+    "dsdk>=1.5.7",
```
Comment (jlubken) [inline {{cookiecutter.name}}/setup.py:8]: Yea!

**Review by jlubken — CHANGES_REQUESTED** [review]
Add dsdk to jupyterlab.

The hope is that mini-conda-forge will find it on pypi, but we;'ll have to try it.

File: {{cookiecutter.name}}/requirements.txt (hunk 1 above)
Comment (GrayEye) [reply {{cookiecutter.name}}/requirements.txt:1]: Added dsdk to requirements

**Review by jlubken — APPROVED** [review]
LGTM

**Pull request merged**